test/specs/behaviors/testDefinitions.ts

```diff
@@ -162,6 +162,49 @@ definitions.push({
   },
 })
 
+// Example:  Adds attribute 'aria-expanded=true' based on the property 'open' if the component has 'hasSubtree' property.
+definitions.push({
+  regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)' based on the property '([a-z]+)' if the component has '([a-zA-Z]+)' property./g,
+  testMethod: (parameters: TestMethod) => {
+    const [
+      attributeToBeAdded,
+      attributeExpectedValue,
+      propertyDependingOnFirst,
+      propertyDependingOnSecond,
+    ] = [...parameters.props]
+
+    const property = {}
+
+    property[propertyDependingOnFirst] = attributeExpectedValue
+    property[propertyDependingOnSecond] = true
+    const expectedResult = parameters.behavior(property).attributes.root[attributeToBeAdded]
+    expect(testHelper.convertToBooleanIfApplicable(expectedResult)).toEqual(
+      testHelper.convertToBooleanIfApplicable(attributeExpectedValue),
+    )
+
+    const propertyFirstPropNegate = {}
+    propertyFirstPropNegate[propertyDependingOnFirst] = !testHelper.convertToBooleanIfApplicable(
+      attributeExpectedValue,
+    )
+    propertyFirstPropNegate[propertyDependingOnSecond] = true
+    const expectedResultFirstPropertyNegate = parameters.behavior(propertyFirstPropNegate)
```

Comment:
Again, a little improvement to naming, to reflect the meaning ```expectedResultFirstPropertyNegate``` - >  ```actualResultFirstPropertyNegate```, because this is what actually the behavior has set.  So then you verify ```actual``` and ```expected```, not ```expected``` and ```expected```.